tools/Status/ExecutableStatus/EvolveGhBinaryBlackHole.py

```diff
@@ -94,3 +97,141 @@ def format(self, field, value):
+                            [
+                                "L2Norm(PointwiseL2Norm(GaugeConstraint))",
+                                "L2Norm(PointwiseL2Norm(ThreeIndexConstraint))",
+                                "L2Norm(PointwiseL2Norm(FourIndexConstraint))",
+                            ]
+                        ],
+                    ],
+                    axis=1,
+                )
+
+        constraints = pd.concat(map(get_constraints_data, reduction_files))
+        constraints.sort_index(inplace=True)
+        fig = px.line(constraints.iloc[1:], log_y=True)
+        fig.update_layout(
+            legend=dict(
+                title=None,
+                orientation="h",
+                yanchor="bottom",
+                y=1,
+                xanchor="left",
+                x=0,
+            )
+        )
+        fig.update_yaxes(exponentformat="e", title=None)
+        st.plotly_chart(fig)
+
+        # Time steps
+        super().render_time_steps(input_file, reduction_files)
+
+        # Trajectories
+        st.subheader("Trajectories")
+        st.pyplot(
+            plot_trajectory(
+                *import_A_and_B(
+                    reduction_files,
+                    "ApparentHorizons/ControlSystemAhA_Centers.dat",
+                    "ApparentHorizons/ControlSystemAhB_Centers.dat",
+                )
+            )
+        )
+
+        # Grid
+        st.subheader("Grid")
+
+        @st.experimental_fragment
+        def render_grid():
+            if st.button("Render grid"):
+                from spectre.Visualization.Render3D.Domain import render_domain
+
+                volume_files = glob.glob(str(run_dir / "BbhVolume*.h5"))
+                generate_xdmf(
+                    volume_files,
+                    output=str(run_dir / "Bbh.xmf"),
+                    subfile_name="VolumeData",
+                )
+                render_domain(
+                    str(run_dir / "Bbh.xmf"),
+                    output=str(run_dir / "domain.png"),
+                    slice=True,
+                    zoom_factor=50.0,
+                    time_step=-1,
+                )
+            if (run_dir / "domain.png").exists():
+                st.image(str(run_dir / "domain.png"))
+
+        render_grid()
+
+        # Control systems
+        st.subheader("Control systems")
+
+        @st.experimental_fragment
+        def render_control_systems():
+            if st.checkbox("Show control systems"):
+                st.pyplot(plot_control_system(reduction_files))
+                with st.expander("Size control A", expanded=False):
+                    st.pyplot(plot_size_control(reduction_files, "A"))
+                with st.expander("Size control B", expanded=False):
+                    st.pyplot(plot_size_control(reduction_files, "B"))
+
+        render_control_systems()
+
+        # Eccentricity
+        # (can be enabled once EccentricityControl is available)
+        # st.subheader("Eccentricity")
```

Comment:
I think you've updated this section since you opened this. Wanna add those changes in?

Reply:
It's enabled now since #5895 was merged